text/0097-log-data-model.md

```diff
@@ -0,0 +1,1340 @@
+  </tr>
+  <tr>
+    <td>errorCode</td>
+    <td>string</td>
+    <td>The AWS service error if the request returns an error.</td>
+    <td>ShortName</td>
+  </tr>
+  <tr>
+    <td>errorMessage</td>
+    <td>string</td>
+    <td>If the request returns an error, the description of the error.</td>
+    <td>Body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>*</td>
+    <td></td>
+    <td>Attributes["cloudtrail.*"]</td>
+  </tr>
+</table>
+
+### Google Cloud Logging
+
+Field            | Type               | Description                                             | Maps to Unified Model Field
+-----------------|--------------------| ------------------------------------------------------- | ---------------------------
+timestamp        | string             | The time the event described by the log entry occurred. | Timestamp
+resource         | MonitoredResource  | The monitored resource that produced this log entry.    | Resource
+log_name         | string             | The URL-encoded LOG_ID suffix of the log_name field identifies which log stream this entry belongs to. | ShortName
+json_payload     | google.protobuf.Struct | The log entry payload, represented as a structure that is expressed as a JSON object. | Body
+proto_payload    | google.protobuf.Any | The log entry payload, represented as a protocol buffer. | Body
+text_payload     | string             | The log entry payload, represented as a Unicode string (UTF-8). | Body
+severity         | LogSeverity        | The severity of the log entry.                          | Severity
+trace            | string             | The trace associated with the log entry, if any.        | TraceId
+span_id          | string             | The span ID within the trace associated with the log entry. | SpanId
+labels           | map<string,string> | A set of user-defined (key, value) data that provides additional information about the log entry. | Attributes
+All other fields |                    |                                                         | Attributes["google.*"]
+
+## Elastic Common Schema
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>@timestamp</td>
+    <td>datetime</td>
+    <td>Time the event was recorded</td>
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td>message</td>
+    <td>string</td>
+    <td>Any type of message</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>labels</td>
+    <td>key/value</td>
+    <td>Arbitrary labels related to the event</td>
+    <td>attributes[*]</td>
+  </tr>
+  <tr>
+    <td>tags</td>
+    <td>array of string</td>
+    <td>List of values related to the event</td>
+    <td>?</td>
+  </tr>
+  <tr>
+    <td>trace.id</td>
+    <td>string</td>
+    <td>Trace ID</td>
+    <td>trace_id</td>
+  </tr>
+  <tr>
+    <td>span.id*</td>
+    <td>string</td>
+    <td>Span ID</td>
+    <td>span_id</td>
+  </tr>
+  <tr>
```

Comment:
I think they should map to TraceId and SpanId

Reply:
Agreed, thanks @okrasz

Reply:
Fixed.